doc/stages/writers.ept_addon.rst

```diff
@@ -0,0 +1,66 @@
+.. _writers.ept_addon:
+
+writers.ept_addon
+=================
+
+The **EPT addon writer** supports writing additional dimensions to `Entwine Point Tile`_ datasets.  Thus the EPT addon writer is unique among most other PDAL writers due to its limitation that it may only be used along with an `EPT reader`_ and modifies and existing dataset rather than creating a brand new one.
+
+The addon dimensions created by this writer may be stored independently from the corresponding EPT dataset, therefore write-access to the EPT resource itself is not required to create and use addon dimensions.
```

Comment:
Not required. Is it possible at all? If so, is it worth documenting how?

Reply:
Really?  Sidecar files?  This seems ugh.

Reply:
The requirement is to be able to augment a tree which you do not have write access. For example, all of the data at usgs.entwine.io.

Reply:
It's not really possible, as there's no EPT level tie-in to these dimensions.  However since they're independent you could place them in a subdirectory of the EPT data if you were so inclined.  I've cleaned up the language a bit to remove the implication that the data might be stored in-band in c57730d.